cvat-ui/src/components/header/header.tsx

```diff
@@ -3,13 +3,23 @@ import React from 'react';
 
 import { RouteComponentProps } from 'react-router';
 import { withRouter } from 'react-router-dom';
+import {
```

Comment:
You do not need this import, right?

Reply:
Yes, I forgot to remove that and the serverInfo files. My bad :(